### PR TITLE
Add RTP/RTCP threadpool relay

### PR DESCRIPTION
**Pull request by feymartynov**

### CODE REVIEW

File: src/conf.rs [hunk 1]
```diff
@@ -41,6 +42,11 @@ pub struct General {
     pub vacuum_interval: i64,
 }
 
+#[derive(Clone, Deserialize, Debug)]
+pub struct Relay {
```
Comment (manifest) [inline src/conf.rs:46]: RelayConfig may be a better name for the struct.

File: src/relay.rs [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+        }
+
+        Self {
+            senders,
+            join_handles: Some(join_handles),
+        }
+    }
+
+    pub fn relay_rtp(&self, session_id: SessionId, packet: &PluginRtpPacket) -> Result<()> {
+        let job = Job::RelayRtp(session_id, OwnedRtpPacket::new(packet));
```
Comment (manifest) [inline src/relay.rs:154]: Do we really need to serialize and deserialize packets?

File: src/conf.rs (hunk 1 above)
Comment (feymartynov) [reply src/conf.rs:46]: What about `Constraint` and `General`?

File: src/relay.rs (hunk 2 above)
Comment (feymartynov) [reply src/relay.rs:154]: Janus frees the memory after `incoming_rtp` callback returns.
We copy the packet to make sure we don't try to access potentially freed buffer in another thread.

File: src/conf.rs (hunk 1 above)
Comment (manifest) [reply src/conf.rs:46]: Maybe I get it wrong. I've thought that the only purpose of that structure is to allow to configure a number of threads. The single public property sort of pointed me in that direction :-)

File: src/relay.rs (hunk 2 above)
Comment (manifest) [reply src/relay.rs:154]: Ok. I see now.

**Pull request closed without merging**